docs/security.md

```diff
@@ -63,10 +65,42 @@ We use a number of static analysis tools to identify security risks in developme
 
 All development laptops have encrypted hard drives.
 
-## Bug reports
+## How to report a security vulnerability
+
+If you think that you have found a security issue, please email us at <a href="mailto:security@sourcegraph.com">security@sourcegraph.com</a>. We will reply to high quality reports within 1 business day to acknowledge that we received them and will strive to send you regular updates on our progress until the issue is resolved. You may request an update by replying to the existing email thread. We will read, but may not respond to or issue bounties for low quality or spammy reports (e.g. those produced by automated tooling).
+
+We provide monetary rewards, up to $10,000 USD, for security vulnerability reports. The actual reward amount is determined based on the number of customers impacted, the difficulty of exploiting the vulnerability, and the severity of the consequences (e.g. service disruption, data leakage, reputational damage to Sourcegraph) of a successful exploit.
+
+We will send payment to a valid PayPal account after the issue is confirmed fixed or 90 days from the original report, whichever happens first. We will ask you for the name and country associated with your PayPal account.
+
+We will not issue a reward if any of the following apply:
+
+1. You engage in disruptive behavior on sourcegraph.com itself (e.g. spamming our system with requests, fake accounts, denial of service). Sourcegraph is [open source software](https://github.com/sourcegraph/sourcegraph), so you can [install a copy yourself](https://docs.sourcegraph.com/#quickstart-guide) and test against that instead.
+1. You publically disclose a vulnerability before we have fixed it.
+1. You spam us with duplicate and/or low quality vulnerability reports (e.g. copy/pasting generic issues from automatic scanning tools).
+1. You are currently a teammate at Sourcegraph (e.g. employee, contractor, intern).
```

Comment:
You may wish to add a family/friend clause here as well?

Reply:
It is a grey area. I added a bullet point here but softened the language at the top: f2e5ee0b7be32bbbfb6e5b1aa1fe885344e47258